config/nunjucks/filters.test.js

```diff
@@ -190,6 +191,22 @@ describe('Nunjucks filters', function() {
         ).to.equal('1 to 10 Nov 2019')
       })
     })
+    context(
+      'when both start date and end date are on the current week ',
+      function() {
+        const weekOpts = {
+          weekStartsOn: 1,
+        }
+        it('returns "this week"', function() {
+          expect(
+            formatDateRange([
+              startOfWeek(new Date(), weekOpts),
+              endOfWeek(new Date(), weekOpts),
+            ])
+          ).to.equal('This week')
```

Comment:
You might want to mock the response of the translation method. If we update that text in the locale this test will fail.